libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -288,7 +288,7 @@ void AP_DDS_Client::update_topic(geometry_msgs_msg_PoseStamped& msg)
     Quaternion orientation;
     if (ahrs.get_quaternion(orientation)) {
         Quaternion aux(orientation[0], orientation[2], orientation[1], -orientation[3]); //NED to ENU transformation
-        Quaternion transformation (sqrt(2)/2,0,0,sqrt(2)/2); // Z axis 90 degree rotation
+        Quaternion transformation (sqrtf(2) * 0.5,0,0,sqrtf(2) * 0.5); // Z axis 90 degree rotation
```

Comment:
sqrtF from AP_MATH

Reply:
Yep, sounds better. AP_MATH distinguishes between boards that have double and single precision and applies `sqrt()` or `sqrtf()` accordingly

```cpp
#if HAL_WITH_EKF_DOUBLE
(...)
#define sqrtF(x) sqrt(x)
(...)
#else
(...)
#define sqrtF(x) sqrtf(x)
```

Thanks for the review! I will change it and push it as a fixup